tasks/60_convert-html-entities/convert-html-entities.js

```diff
@@ -0,0 +1,18 @@
+export function convertHTML(str) {
+  const signs = {
+    '&': '&amp;',
+    '<': '&lt;',
+    '>': '&gt;',
+    '"': '&quot;',
+    '\'': '&apos;',
+  };
+  [...str].forEach((letter) => {
```

Comment:
it can be done much easier with just replace with function as the second argument which will receive matched symbol and you will just take it from object signs[matchedSymbol]